src/v1/types/responses.rs

```diff
@@ -1,27 +1,216 @@
 //! Responses from a service.
 
+use std::collections::BTreeMap;
+
+use chrono::DateTime;
+use chrono::Utc;
+
+use crate::v1::types::task::Executor;
+use crate::v1::types::task::Input;
+use crate::v1::types::task::Output;
+use crate::v1::types::task::Resources;
+use crate::v1::types::task::State;
+
 pub mod service_info;
-pub mod task;
 
 pub use service_info::ServiceInfo;
 
 /// A response from `POST /tasks`.
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-#[cfg_attr(feature = "ord", derive(Ord, PartialOrd))]
-pub struct CreateTask {
+pub struct CreatedTask {
     /// The ID of the created task.
     pub id: String,
 }
 
 /// The response from `GET /tasks`.
 #[derive(Clone, Debug, Default, Eq, PartialEq)]
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
-#[cfg_attr(feature = "ord", derive(Ord, PartialOrd))]
 pub struct ListTasks<Task> {
     /// The tasks in this page of results.
     pub tasks: Vec<Task>,
 
     /// The token for the next page of results.
     pub next_page_token: Option<String>,
 }
+
+/// A task output file.
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct OutputFile {
+    /// The URL.
+    pub url: String,
+
+    /// The path within the container.
+    pub path: String,
+
+    /// The size in bytes.
+    pub size_bytes: String,
+}
+
+/// A log for an [`Executor`].
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
+#[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
+pub struct ExecutorLog {
+    /// The start time.
+    pub start_time: Option<DateTime<Utc>>,
+
+    /// The end time.
+    pub end_time: Option<DateTime<Utc>>,
+
+    /// The value of the standard output stream.
+    pub stdout: Option<String>,
+
+    /// The value of the standard error stream.
+    pub stderr: Option<String>,
+
+    /// The exit code.
+    pub exit_code: Option<u32>,
```

Comment:
It appears that this is a required field.

Reply:
Already pushed up.